homeassistant/components/huisbaasje/sensor.py

```diff
@@ -0,0 +1,255 @@
+                device_class=DEVICE_CLASS_POWER,
+                source_type=SOURCE_TYPE_ELECTRICITY,
+            ),
+            HuisbaasjeSensor(
+                coordinator,
+                user_id=user_id,
+                name="Electricity In Rate",
+                device_class=DEVICE_CLASS_POWER,
+                source_type=SOURCE_TYPE_ELECTRICITY_IN,
+            ),
+            HuisbaasjeSensor(
+                coordinator,
+                user_id=user_id,
+                name="Electricity In Low Rate",
+                device_class=DEVICE_CLASS_POWER,
+                source_type=SOURCE_TYPE_ELECTRICITY_IN_LOW,
+            ),
+            HuisbaasjeSensor(
+                coordinator,
+                user_id=user_id,
+                name="Electricity Out Rate",
+                device_class=DEVICE_CLASS_POWER,
+                source_type=SOURCE_TYPE_ELECTRICITY_OUT,
+            ),
+            HuisbaasjeSensor(
+                coordinator,
+                user_id=user_id,
+                name="Electricity Out Low Rate",
+                device_class=DEVICE_CLASS_POWER,
+                source_type=SOURCE_TYPE_ELECTRICITY_OUT_LOW,
+            ),
+            HuisbaasjeSensor(
+                coordinator,
+                user_id=user_id,
+                name="Electricity Today",
+                unit_of_measurement=ENERGY_KILO_WATT_HOUR,
+                source_type=SOURCE_TYPE_ELECTRICITY,
+                sensor_type=SENSOR_TYPE_THIS_DAY,
+                icon="mdi:counter",
+            ),
+            HuisbaasjeSensor(
+                coordinator,
+                user_id=user_id,
+                name="Gas Rate",
+                unit_of_measurement=FLOW_CUBIC_METERS_PER_HOUR,
+                source_type=SOURCE_TYPE_GAS,
+                icon="mdi:fire",
+            ),
+            HuisbaasjeSensor(
+                coordinator,
+                user_id=user_id,
+                name="Gas Today",
+                unit_of_measurement=VOLUME_CUBIC_METERS,
+                source_type=SOURCE_TYPE_GAS,
+                sensor_type=SENSOR_TYPE_THIS_DAY,
+                icon="mdi:counter",
+            ),
+        ]
+    )
+
+
+def _get_measurement_rate(current_measurements: dict, source_type: str):
+    if source_type in current_measurements.keys():
+        if current_measurements[source_type]["measurement"]:
+            return current_measurements[source_type]["measurement"]["rate"]
+    else:
+        _LOGGER.warn(
+            f"Source type '{source_type}' not present in {current_measurements}"
+        )
+    return None
+
+
+def _get_this_day_value(current_measurements: dict, source_type: str):
+    if source_type in current_measurements.keys():
+        if current_measurements[source_type]["thisDay"]:
+            return current_measurements[source_type]["thisDay"]["value"]
+    else:
+        _LOGGER.warn(
+            f"Source type '{source_type}' not present in {current_measurements}"
```

Comment:
Why do you use quotes around {source_type}? They are not really needed.
Also, please use_LOGGER.debug or .error depending on the severity of this message.